src/styles/Button.css

```diff
@@ -1,9 +1,7 @@
-.generalBtn {
-	width: 100px;
-	height: 60px;
+.normal {
```

Comment:
The button component should have borders ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/ButtonsBoarder.png)